network/f5/bigip_gtm_pool.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2015, Michael Perzel
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: bigip_gtm_pool
+short_description: "Manages F5 BIG-IP GTM pools"
+description:
+    - "Manages F5 BIG-IP GTM pools"
+version_added: "1.9"
+author: 'Michael Perzel'
+notes:
+    - "Requires BIG-IP software version >= 11.4"
+    - "F5 developed module 'bigsuds' required (see http://devcentral.f5.com)"
+    - "Best run as a local_action in your playbook"
+    - "Tested with manager and above account privilege level"
+
+requirements:
+    - bigsuds
+options:
+    server:
+        description:
+            - BIG-IP host
+        required: true
+        default: null
+        choices: []
+        aliases: []
+    user:
+        description:
+            - BIG-IP username
+        required: true
+        default: null
+        choices: []
+        aliases: []
+    password:
+        description:
+            - BIG-IP password
+        required: true
+        default: null
+        choices: []
+        aliases: []
+    state:
+        description:
+            - Pool member state
+        required: true
+        choices: ['present', 'absent', 'enabled', 'disabled']
+        aliases: []
+    pool:
+        description:
+            - Pool name
+        required: true
+        default: null
+        aliases: []
+    partition:
+        description:
+            - Partition name
+        required: false
+        default: Common
+        aliases: []
+'''
+
+EXAMPLES = '''
+  - name: Disable pool
+    local_action: >
+      bigip_gtm_pool
+      server=192.168.0.1
+      user=admin
+      password=mysecret
+      state=disabled
+      pool=my_pool
+      partition=Common
+'''
+try:
+    import bigsuds
+except ImportError:
+    bigsuds_found = False
+else:
```

Comment:
no need for else, you can set to true under the import, if exception happens it will be set explicitly to false (just a style note, no need to change).